README.md

```diff
@@ -52,3 +52,14 @@ $ export ETH_GAS_LIMIT=8000000
 $ export ETH_GAS_PRICE=$(seth --to-wei 3 "gwei")
 $ make deploy
 ```
+
+### Cast to tenderly
+
+1. Create Tenderly account (no trial period needed atm) https://dashboard.tenderly.co/register
+    - Note down `TENDERLY_USER` and `TENDERLY_PROJECT` values
```

Comment:
Nit: this reads like these values will only be accessible if you're creating the user. I would adjust this instruction to say something along the lines:

```
- get your username from https://dashboard.tenderly.co/account and note it down for later use as `TENDERLY_USER` env variable
- get your tenderly project name from https://dashboard.tenderly.co/projects and note it down to use later as `TENDERLY_PROJECT` env variable
```

Reply:
Thanks for providing extra instructions, but I would try to avoid writing a full tutorial on how to setup tenderly account, because we don't want to maintain it (and URLs and UIs tend to change over time)